src/Discovery/src/ClientCore/SimpleClients/NoOpDiscoveryClient.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Logging;
+using Steeltoe.Common;
+using Steeltoe.Common.Discovery;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Steeltoe.Discovery.Client
+{
+    internal class NoOpDiscoveryClient : IDiscoveryClient
+    {
+        internal NoOpDiscoveryClient(ILogger<NoOpDiscoveryClient> logger = null)
+        {
+            logger?.LogWarning("A NoOp Discovery Client has been created! Running in container: {IsContainerized}", Platform.IsContainerized);
+        }
+
+        public string Description => "The IDiscoveryClient that didn't";
```

Comment:
I don't think this is a good description.

Reply:
Fair point. I wasn't taking it very seriously because I don't think description is actually used anywhere, but we could do something like this
```suggestion
        public string Description => "An IDiscoveryClient that passes through to underlying infrastructure";
```

Reply:
Much better.